### PR TITLE
Api provider changed 

### PR DESCRIPTION
**Pull request by stemmr**
also removed local dependency

### CODE REVIEW

**Review by jacogr — APPROVED** [review]
LGTM

File: src/index.js [hunk 1]
```diff
@@ -1,44 +1,28 @@
...
-	// Fallback to localhost:8545
-	if (!transport) {
-		transport = new Parity.Api.Transport.Http('http://localhost:8545');
-	}
-
-	return transport;
+function defaultProvider () {
+  	if(typeof window !== 'undefined' && window.ethereum){	
+  		return window.ethereum;
+  	}else if(typeof ParityApi !=='undefined'){
```
Comment (jacogr) [inline src/index.js]: Would keep the original top-level if as the else-if - that way it works both way around, i.e.

```
if (typeof window !== 'undefined' && window.ethereum) {
 ...
} else if (typeof(window) !== 'undefined' && typeof(window.parity) !== 'undefined' && window.parity.api.transport._url) {
}...
```

In addition typeof ParityApi will never be undefined, you include it.

Comment (gavofyork) [inline src/index.js]: please keep the original whitespace style.

File: src/index.js [hunk 2]
```diff
@@ -1,44 +1,28 @@
...
+  		return new ParityApi.Provider.Http('http://localhost:8545');
+  	}else{
+  		throw 'Could not connect to provider, please check connection';
+  	}
 }
 
-export function Bonds(transport = defaultTransport()) {
-	return createBonds({ api: new Parity.Api(transport) });
+function Bonds (provider = defaultProvider()) {
+  return createBonds({ api: new ParityApi(provider) });
```
Comment (gavofyork) [inline src/index.js]: tabs here, not spaces.

**Review by gavofyork — CHANGES_REQUESTED** [review]
Reasonable after from the whitespace.

File: src/index.js (hunk 1 above)
Comment (jacogr) [reply src/index.js]: We need to get the eslint stuff in here ASAP, I'll split it out from my earlier stuff in smaller chunks. (Along with .editorconfig, so there things just won't pop up.)

**Review by gavofyork — APPROVED** [review]

**Pull request merged**